cqproto/grpc.go

```diff
@@ -102,13 +109,20 @@ func (g *GRPCServer) GetProviderConfig(ctx context.Context, _ *internal.GetProvi
 }
 
 func (g *GRPCServer) ConfigureProvider(ctx context.Context, request *internal.ConfigureProvider_Request) (*internal.ConfigureProvider_Response, error) {
+
+	var eFields = make(map[string]interface{})
+	if err := json.Unmarshal(request.GetExtraFields(), &eFields); err != nil {
```

Comment:
Will `GetExtraFields` ever return `nil`? If so, I think this will panic here.

Reply:
https://play.golang.org/p/dyqF3cZMWOd
won't panic but will return an error which is not the flow we want.
fixed to check for nil